packages/shared/src/lib/analytics.ts

```diff
@@ -132,6 +132,8 @@ export enum AnalyticsEvent {
   GenerateDevcard = 'generate devcard',
   DownloadDevcard = 'download devcard',
   CopyDevcardCode = 'copy devcard code',
+  // 404 page
+  View404Page = '404 page',
```

Comment:
Are we fixing 404 on profile page that we mentioned or is that out of scope for now?

Reply:
The change is out of scope, this will track all implementations of 404 showing though.
Regardless of post 404 or profile one :)